torch/_inductor/codecache.py

```diff
@@ -1361,6 +1365,7 @@ def get_remote_cache() -> Optional[RemoteCache[JsonDataTy]]:
     @staticmethod
     def load_with_key(
         key: str,
+        gm: torch.fx.GraphModule,
```

Comment:
@masnesral Sorry, the whole point of load_with_key is that you don't pass the `gm` in either haha, do you mind threading it through here too? 

I know the gm is only used for constants here, but it's called in AOTAutogradCache. 

I think in my head, the way it works is taht both AOTAutogradCache.load() and FXGraphCache.load() calculate constants off of their respective gms and then call FXGraphCache.load_with_key()

Reply:
@jamejwu, sorry I guess I'm probably not really following what you're trying to achieve. The primary change in this PR is to stop storing the constants in the cache entry. i.e., we don't have the constants available when we create the module here: https://github.com/pytorch/pytorch/blob/08ce3aac6222bc044a34289be0885e37229d98a2/torch/_inductor/codecache.py#L1124

Instead, when we synthesize the module from the cache, we retrieve the constants from the GraphModule. So I don't understand how we can not pass the GraphModule on the cache lookup path. We can't extract the constants first, because we won't know the attribute _names_ until we have a cache hit and get the names from the cache entry. If the goal is to pass the bare minimum, then I think we'd need to just pass all of the GraphModule's attributes instead of the whole GraphModule object. But that looks a bit strange to me. Lemme know if I totally misunderstand your ask.